Pod/Classes/RZMessagingWindow.m

```diff
@@ -288,4 +289,84 @@ - (void)configurePassThroughView
 #pragma mark - RZRootMessagingViewController
 
 @implementation RZRootMessagingViewController
+
+#pragma mark - Orientation method overrides
+
+-(BOOL)shouldAutorotate
+{
+    UIViewController *topViewController = [RZRootMessagingViewController topViewController];
+    
+    if ( topViewController == self ) {
+        // If the top view controller is itself then do the same as
+        // default by returning YES.
+        return YES;
+    }
+    else {
+        return topViewController.shouldAutorotate;
+    }
+}
+
+- (NSUInteger)supportedInterfaceOrientations
+{
+    UIViewController *topViewController = [RZRootMessagingViewController topViewController];
+    
+    if ( topViewController == self ) {
+        // If the top view controller is itself then do the same as
+        // default by:
+        // 1)For iPhone, all except upside down
```

Comment:
I skipped upside down to match default behavior of view controllers.  The doc for supportedInterfaceOrientations says that iPhone and iPad have different values.  I would be happy to change it if you rather have just consistently return mask all.  Please let me know.

![image](https://cloud.githubusercontent.com/assets/9040031/5784892/8cd9cd68-9d85-11e4-89b9-4986ab1ae615.png)